### PR TITLE
Add spotify-recent-plays connector

### PR DESCRIPTION
**Pull request by Bajena**
This connector queries recently played tracks from [Spotify API](https://developer.spotify.com/documentation/web-api/reference/player/get-recently-played/).

The whole development process has been described in this medium series:
https://medium.com/@bajena3/building-a-custom-google-data-studio-connector-from-a-z-b4d711a5cf58

The main repo can be found here: https://github.com/Bajena/spotify-gds-connector

Here are the current CI & CodeCov statuses for the connector:
[CircleCI](https://circleci.com/gh/Bajena/spotify-gds-connector/tree/master)
[codecov](https://codecov.io/gh/Bajena/spotify-gds-connector)

Hope it's useful to someone :)

Preview:
image

### CODE REVIEW

**Comment by Bajena** [comment]
I signed it!

File: spotify-recent-plays/src/Connector.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+ */
+Connector.prototype.getAuthType = function() {
+  return { type: 'OAUTH2' };
+};
+
+/**
+ * @return {boolean}
+ */
+Connector.prototype.isAdminUser = function() {
+  return true;
```
Comment (diminishedprime) [inline spotify-recent-plays/src/Connector.js]: isAdminUser should either return false, or do some sort of check to determine if the current user of the connector is actually the admin.

File: spotify-recent-plays/test/Connector.test.js [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+      {
+        "dataType": "STRING",
+        "label": "Artist",
+        "name": "artist",
+        "semantics": {
+          "conceptType": "DIMENSION"
+        }
+      }
+    ]
+  };
```
Comment (diminishedprime) [inline spotify-recent-plays/test/Connector.test.js]: Could you par down this test data a bit, it's size sorta gets in the way of what it's testing.

File: spotify-recent-plays/src/Connector.js [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+ * @return {object} Object containing auth config
+ */
+Connector.prototype.getAuthType = function() {
+  return { type: 'OAUTH2' };
+};
+
+/**
+ * @return {boolean}
+ */
+Connector.prototype.isAdminUser = function() {
```
Comment (diminishedprime) [inline spotify-recent-plays/src/Connector.js]: This should return false, or have logic that actually checks if the current user is the admin of the connector.

**Comment by Bajena** [comment]
Thanks for the review @diminishedprime! I applied your suggestions 👍

**Comment by Bajena** [comment]
Regarding `yarn prettier` suggestion. I'm using a set of eslint rules. You can run `yarn lint` to verify that the code is correctly formatted.

**Comment by Bajena** [comment]
@diminishedprime Could you take a look once again?

**Comment by diminishedprime** [comment]
Thanks, the changes look good!

**Pull request merged**